pkg/nat/nat.go

```diff
@@ -84,6 +102,14 @@ func (p Port) Int() int {
 	return int(port)
 }
 
+// Range returns the start/end port numbers of a Port range as ints
+func (p Port) Range() (int, int) {
+	// We don't need to check for an error because we're going to
+	// assume that any error would have been found, and reported, in NewPort()
```

Comment:
@mavenugo I agree.  This was just following the pattern given in Port.Int(), which ignores errors with the same assumption.  I will add the error case